index/server/pkg/server/index.go

```diff
@@ -72,6 +73,7 @@ func ServeRegistry() {
 		Handler:      handler,
 		ReadTimeout:  10 * time.Second,
 		WriteTimeout: 10 * time.Second,
+		TLSNextProto: make(map[string]func(*http.Server, *tls.Conn, http.Handler)),
```

Comment:
Similar to what was done in https://github.com/devfile/registry-operator/pull/60, should we have an option for the registry administrator to enable http/2 and disable by default?

Reply:
Oh nice idea. I'll try to add the same flow so we can be consistent :) thanks @michael-valdron

Reply:
@michael-valdron I've added the `ENABLE_HTTP2` variable and tried to follow the same approach as the operator.